Service/Runner.php

```diff
@@ -158,7 +159,8 @@ private function createContainerFromImage(
         RunCommandOptions $runCommandOptions,
         WorkingDirectory $workingDirectory,
         OutputFilterInterface $outputFilter,
-        Limits $limits
+        Limits $limits,
+        array $features
```

Comment:
zaprve to tomu nepredavas v runImages, a za druhe je to zbytecny 
v Container se ke componente dostanes pres $this->getImage()->getSourceComponent()